cmd/composer-cli/blueprints/save.go

```diff
@@ -49,7 +49,7 @@ func saveToml(cmd *cobra.Command, args []string) (rcErr error) {
 	for _, bp := range bps {
 		name, ok := bp.(map[string]interface{})["name"].(string)
 		if !ok {
-			fmt.Fprintf(os.Stderr, "ERROR: no 'name' in blueprint\n")
+			fmt.Fprintf(os.Stderr, "ERROR: no '%s' in blueprint\n", name)
```

Comment:
This won't work. What this block is doing is testing to see if there is a 'name' field in the json data. name will always be an empty string when ok is false. ok indicates whether or not it was in the map.

Reply:
Opps, I thought this was the blueprint name. Reverted.